tgui/packages/tgui/interfaces/AntagInfoMalf.tsx

```diff
@@ -101,28 +101,28 @@ function FlavorSection(props) {
           <Stack fill vertical>
             <Stack.Item style={{ backgroundColor: 'black' }}>
               <span style={goalstyle}>
-                System Integrity Report:
+                Отчет о целостности системы:
                 <br />
               </span>
               &gt;{goal}
             </Stack.Item>
             <Stack.Divider />
             <Stack.Item grow style={{ backgroundColor: 'black' }}>
               <span style={allystyle}>
-                Morality Core Report:
+                Доклад ядра морали:
```

Comment:
"Во всех системах станций обнаружены вредоносные процессы. Пожалуйста, уничтожьте свой ИИ, чтобы предотвратить возможный ущерб его моральному ядру."
```suggestion
                Доклад морального ядра:
```